<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="C:\Users\user\Documents\GitHub\PersonalBudgeting\PersonalBudgeting.Tests\bin\Debug\PersonalBudgeting.Tests.dll" total="1" errors="0" failures="1" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2014-06-19" time="15:11:43">
  <environment nunit-version="2.6.3.13283" clr-version="2.0.50727.5477" os-version="Microsoft Windows NT 6.1.7601 Service Pack 1" platform="Win32NT" cwd="C:\Program Files (x86)\NUnit 2.6.3\bin" machine-name="PBIO-2258" user="User" user-domain="PBIO-2258" />
  <culture-info current-culture="en-US" current-uiculture="en-US" />
  <test-suite type="Assembly" name="C:\Users\user\Documents\GitHub\PersonalBudgeting\PersonalBudgeting.Tests\bin\Debug\PersonalBudgeting.Tests.dll" executed="True" result="Failure" success="False" time="0.455" asserts="0">
    <results>
      <test-suite type="Namespace" name="PersonalBudgeting" executed="True" result="Failure" success="False" time="0.384" asserts="0">
        <results>
          <test-suite type="Namespace" name="Tests" executed="True" result="Failure" success="False" time="0.383" asserts="0">
            <results>
              <test-suite type="TestFixture" name="FunctionalityTests" executed="True" result="Failure" success="False" time="0.374" asserts="0">
                <results>
                  <test-case name="PersonalBudgeting.Tests.FunctionalityTests.TestForThreePaysAndFourWithdrawals" executed="True" result="Failure" success="False" time="0.217" asserts="1">
                    <failure>
                      <message><![CDATA[  Expected: 13190.299999999999d +/- 0.10000000000000001d
  But was:  1546.9236240937144d
]]></message>
                      <stack-trace><![CDATA[at PersonalBudgeting.Tests.FunctionalityTests.TestForThreePaysAndFourWithdrawals() in c:\Users\user\Documents\GitHub\PersonalBudgeting\PersonalBudgeting.Tests\FunctionalityTests.cs:line 59
]]></stack-trace>
                    </failure>
                  </test-case>
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>